src/ansible_navigator/configuration_subsystem/configurator.py

```diff
@@ -234,7 +234,17 @@ def _apply_cli_params(self) -> None:
                 self._messages.append(LogMessage(level=logging.INFO, message=message))
 
     def _post_process(self) -> None:
+        delayed = []
+        normal = []
```

Comment:
```suggestion
        delayed_config_entries = []
        normal_config_entries = []
```
(maintainability+readability)

Reply:
In this case I don't have strong feelings about the variable names here since they are limited to the function.

Reply:
I have a preference for seeing a "delayed config entries list" vs a "list of unspecified delayed abstractions".